dev-helpers/template.ejs

```diff
@@ -5,7 +5,6 @@
   <meta charset="UTF-8">
```

Comment:
This was changed to ejs template for my previous approach and I forgot to revert it. Now index.html is back.